data/json/items/containers.json

```diff
@@ -443,7 +443,7 @@
     "description": "A small cardboard box.  No bigger than a foot in dimension.",
     "weight": "151 g",
     "volume": "1 L",
-    "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "990 ml", "max_contains_weight": "4 kg" } ],
+    "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": false, "max_contains_volume": "990 ml", "max_contains_weight": "4 kg" } ],
```

Comment:
I feel like cardboard boxes should be rigid, but I guess that's arguable.  If it's not rigid then I think a volume of 1L is excessive.

(Similar comments apply to the other boxes below).

Reply:
the rigid thing is because of magazine_well, and it's meant to keep the previous behavior of "this is the volume of a folded cardboard box". it's reasonable to audit them too, but the rigid vs. magazine_well behavior was obviously wrong.

Reply:
This one doesn't have a magazine well.  I guess it should, to match the other two similar boxes?

Reply:
If `magazine_well` requires `rigid: false`, is that also being checked?